ddp-workspace/projects/toolkit-prion/src/lib/components/activity/prionActivity.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, Inject, OnInit } from '@angular/core';
+import { ActivityComponent, ToolkitConfigurationService, WorkflowBuilderService } from 'toolkit';
+import { ActivatedRoute } from '@angular/router';
+
+@Component({
+  selector: 'prion-activity',
+  template: `
+    <ng-container>
+        <prion-header>
+        </prion-header>
+        <div class="ContainerSurvey-top row">
+        <ddp-activity [studyGuid]="studyGuid"
+                      [activityGuid]="id"
+                      (submit)="navigate($event)"
+                      class="ContainerSurvey col-lg-8 col-lg-offset-2 col-md-8 col-md-offset-2 col-sm-10 col-sm-offset-1 col-xs-12">
+        </ddp-activity>
+        </div>
+    </ng-container>`
+})
+export class PrionActivityComponent extends ActivityComponent  {
+  public id: string;
+  public studyGuid: string;
+
+  constructor(
+    private _activatedRoute: ActivatedRoute,
+    private _workflowBuilder: WorkflowBuilderService,
+    @Inject('toolkit.toolkitConfig') private _toolkitConfiguration: ToolkitConfigurationService) {
+    super(_activatedRoute, _workflowBuilder, _toolkitConfiguration);
+  }
+}
```

Comment:
These two components (prionActivity.component.ts and prionActivityPage.component.ts) look awfully similar. Please review to ensure that you indeed need both.